src/aurora/fevfile.h

```diff
@@ -0,0 +1,131 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  A loader for FEV (FMOD Event) files.
+ */
+
+#ifndef AURORA_FEVFILE_H
+#define AURORA_FEVFILE_H
+
+#include "src/common/readstream.h"
+#include "src/common/ustring.h"
+
+namespace Aurora {
+
+/** Possible types of streaming for fmod. */
+enum FEVStreamingType {
+	kDecompressIntoMemory,
+	kLoadIntoMemory,
+	kStreamFromDisk
+};
+
+/** Reference to an external wave bank. */
+struct FEVWaveBank {
+	uint32 maxStreams;
+	FEVStreamingType streamingType;
+	Common::UString name;
+};
+
+/** A category which is organized hierarchically. */
+struct FEVCategory {
+	Common::UString name;
+	uint32 volume;
+	uint32 pitch;
+};
+
+/**
+ * An fmod event.
+ *
+ * @note
+ * 	Most of the floating point values only represent a
+ * 	range from 0.0f to 1.0f, which is mapped to different
+ * 	dezibel values.
+ */
+struct FEVEvent {
```

Comment:
Please, put these structs and enums into class FEVFile, as FEVFile::Category, etc